config/envoyconfig/listeners_grpc.go

```diff
@@ -64,6 +64,7 @@ func (b *Builder) buildGRPCHTTPConnectionManagerFilter() *envoy_config_listener_
 		"databroker.DataBrokerService",
 		"registry.Registry",
 		"grpc.health.v1.Health",
+		"grpc.reflection.v1alpha.ServerReflection",
```

Comment:
Is this used by a particular client? Only asking since there are two versions of this API and some clients can use both at the same time.